week.yaml

```
@@ -1,40 +1,40 @@
-    job_title_or_ref: "Senior .Net Dev"
-    employer: "Cascade Data Systems"
+  - date: "09/23/2025"
+    activity_choice: "employer contact"
+    job_title: ".NET Engineer"
+    business_name: "Cascade Data Systems"
     address: "9800 Rainier Ave S"
     city: "Seattle"
     state: "WA"
     website_or_email: "https://www.cascadedata.io"
     phone: "(206) 555-9876"
+    contact_method: [ "Online" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)